tests/moduletool/test_install.py

```diff
@@ -120,54 +143,84 @@ def test_install_for_git_failures(modules_dir, modules_repo):
+        ["git", "clone", os.path.join(git_modules_dir, "repos", "noverproject")], cwd=git_modules_dir, stderr=subprocess.STDOUT
     )
     os.chdir(coroot)
     Config.load_config()
 
-    ModuleTool().execute("install", [])
+    ProjectTool().execute("install", [])
 
 
-def test_bad_dep_checkout(modules_dir, modules_repo):
-    coroot = os.path.join(modules_dir, "baddep")
+def test_bad_dep_checkout(git_modules_dir, modules_repo):
+    coroot = os.path.join(git_modules_dir, "baddep")
     subprocess.check_output(
-        ["git", "clone", os.path.join(modules_dir, "repos", "baddep")], cwd=modules_dir, stderr=subprocess.STDOUT
+        ["git", "clone", os.path.join(git_modules_dir, "repos", "baddep")], cwd=git_modules_dir, stderr=subprocess.STDOUT
     )
     os.chdir(coroot)
     Config.load_config()
 
     with pytest.raises(CompilerException):
-        ModuleTool().execute("install", [])
+        ProjectTool().execute("install", [])
 
 
-def test_master_checkout(modules_dir, modules_repo):
-    coroot = install_project(modules_dir, "masterproject")
+def test_master_checkout(git_modules_dir, modules_repo):
+    coroot = install_project(git_modules_dir, "masterproject")
 
-    ModuleTool().execute("install", [])
+    ProjectTool().execute("install", [])
 
     dirname = os.path.join(coroot, "libs", "mod8")
     assert os.path.exists(os.path.join(dirname, "devsignal"))
     assert os.path.exists(os.path.join(dirname, "mastersignal"))
 
 
-def test_dev_checkout(modules_dir, modules_repo):
-    coroot = os.path.join(modules_dir, "devproject")
+def test_dev_checkout(git_modules_dir, modules_repo):
+    coroot = os.path.join(git_modules_dir, "devproject")
     subprocess.check_output(
-        ["git", "clone", os.path.join(modules_dir, "repos", "devproject")], cwd=modules_dir, stderr=subprocess.STDOUT
+        ["git", "clone", os.path.join(git_modules_dir, "repos", "devproject")], cwd=git_modules_dir, stderr=subprocess.STDOUT
     )
     os.chdir(coroot)
     Config.load_config()
 
-    ModuleTool().execute("install", [])
+    ProjectTool().execute("install", [])
 
     dirname = os.path.join(coroot, "libs", "mod8")
     assert os.path.exists(os.path.join(dirname, "devsignal"))
     assert not os.path.exists(os.path.join(dirname, "mastersignal"))
+
+
+@pytest.mark.parametrize("editable", [True, False])
+@pytest.mark.parametrize("set_path_argument", [True, False])
+def test_module_install(tmpdir: py.path.local, modules_dir: str, editable: bool, set_path_argument: bool) -> None:
+    module_path: str = os.path.join(modules_dir, "minimalv2module")
+    python_module_name: str = "inmanta-module-minimalv2module"
+    venv.create(tmpdir, with_pip=True)
+    pip: str = os.path.join(tmpdir, "bin", "pip")
+    # default pip version is not compatible with module install flow
+    subprocess.check_output([pip, "install", "-U", "pip"])
+
+    def is_installed(name: str, only_editable: bool = False) -> None:
```

Comment:
```suggestion
    def is_installed(name: str, only_editable: bool = False) -> bool:
```

Reply:
Thanks for this. This reminds me I forgot to do a second mypy check.